src/phoenix/server/main.py

```diff
@@ -189,6 +189,7 @@ def _get_pid_file() -> Path:
 
     # If the host is "::", the convention is to bind to all interfaces. However, uvicorn
     # does not support this directly unless the host is set to None.
+    display_host = host
     if ":" in host:
         # format IPv6 hosts in brackets
         display_host = f"[{host}]"
```

Comment:
Was `mypy` not catching this?

Reply:
I think we are merging without checking CI?

Reply:
Must be.